reactnative/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "react-native-text-mask",
+  "version": "0.18.0",
+  "description": "React native input component that accepts mask pattern",
```

Comment:
The "n" in "native" should be capital.

Also, I been meaning to change the description of these components to something more direct and SEO friendly like:

"React Native input masking component"